integration-tests/kotlin-dsl/pom.xml

```diff
@@ -39,6 +39,30 @@
             <groupId>org.apache.camel.quarkus</groupId>
             <artifactId>camel-quarkus-direct</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.camel.quarkus</groupId>
+            <artifactId>camel-quarkus-seda</artifactId>
```

Comment:
Please regenerate the `virtualDependencies` profile if the corresponding `-deployment` dependencies are not present.

Reply:
You make me realize that I launched the command from the wrong module, thx for the feedback